modules/oe_whitelabel_list_pages/tests/src/Functional/ListPagesTest.php

```diff
@@ -72,36 +109,81 @@ public function testListPage(): void {
     $list_page->get('emr_entity_metas')->attach($list_page_entity_meta);
     $list_page->save();
 
-    $this->drupalGet('node/' . $list_page->id());
+    return $list_page;
+  }
 
-    // Assert the left column.
-    $assert_session->elementExists('css', 'div.row > div.col-12.col-lg-3');
-    // Assert offcanvas.
-    $assert_session->elementExists('css', 'div.bcl-offcanvas');
-    $assert_session->elementTextEquals('css', 'h4.offcanvas-title', 'Filter options');
-    $assert_session->elementExists('css', 'button.btn-light > svg');
-    $assert_session->elementTextEquals('css', 'button[data-bs-toggle="offcanvas"]', 'Filters');
-    // Assert Filters.
-    $assert_session->elementExists('css', 'input[name="oe_sc_news_title"]');
-    $assert_session->elementExists('css', 'input[data-drupal-selector="edit-submit"]');
-    $assert_session->elementExists('css', 'input[data-drupal-selector="edit-reset"]');
+  /**
+   * Asserts the title in the facets summary.
+   *
+   * @param int $expected_count
+   *   Expected number of results to be reported in the title.
+   * @param \Behat\Mink\Element\ElementInterface $container
+   *   Document to check against.
+   */
+  protected function assertFacetsSummaryTitle(int $expected_count, ElementInterface $container): void {
+    $facets_summary = $container->find('css', 'div.col-md-6.col-lg-8');
+    $title = $facets_summary->find('css', 'h4');
```

Comment:
Can be simplified:
```suggestion
    $title = $container->find('css', 'div.col-md-6.col-lg-8 h4');
```
Also, I wonder if we want to be so detailed with the selectors.
I think it's ok because this is our template, we are adding the classes. They are not coming from bcl. When we change them, we should change the test as well.
But then the question is how far do we want to go with this? E.g. would we also want to test for `'.text-capitalize'`?

Btw in newest netlify / bcl this has changed again. (assuming that what we see here was copied from bcl at some point)

Reply:
my compass is to leave the ones that are not utility related but structure/layout related, so I dont add mt-4 or ps-0 but I add col-12 and such.
Its a bit loose as if I add the classes myself then I might add more classes. Not 100% convinced of the best way neither..